sweetalert2.d.ts

```diff
@@ -422,7 +401,26 @@ declare module 'sweetalert2' {
     | 'title'
     | 'titleText';
 
-  export type SweetAlertArrayOptions = [string?, string?, SweetAlertIcon?];
+  export interface SweetAlertCustomClass extends SweetAlertCommonClass {
```

Comment:
A question that came to my mind when I was working on this... does `update` support "patch" update?

For example...

```ts
swal.fire({
  title: 'blabla',
  customClass: {
    popup: 'my-popup'
  }
})

swal.update({
  customClass: {
    backdrop: 'my-backdrop'
  }
})
```

... the `customClass` will be `{ popup: 'my-popup', backdrop: 'my-backdrop' }` or just `{ backdrop: 'my-backdrop' }`?

Reply:
No, "patch" updates aren't supported.

Reply:
@rafaelss95 The `update` method doesn't do "patch" updates by itself, but you can accomplish this using the `params` member. Example:

```js
const swal = Swal.fire({
  customClass: { popup: 'my-popup' }
})
swal.update({
  customClass: { ...swal.params, backdrop: 'my-backdrop' }
})
console.log(swal.params.customClass) // { popup: 'my-popup', backdrop: 'my-backdrop' }
```

Reply:
@zenflow Yes, I'm aware of that way :)